codegen-core/src/main/kotlin/software/amazon/smithy/rust/codegen/core/smithy/customizations/SmithyTypesPubUseExtra.kt

```diff
@@ -79,4 +80,16 @@ fun pubUseSmithyPrimitives(codegenContext: CodegenContext, model: Model, rustCra
             "SdkBody" to RuntimeType.smithyTypes(rc).resolve("body::SdkBody"),
         )
     }
+    if (codegenContext.serviceShape.hasEventStreamOperations(model)) {
+        rustTemplate(
+            """
+            pub use #{Header};
+            pub use #{HeaderValue};
+            pub use #{Message};
```

Comment:
Do we also want to reexport `StrBytes` conditionally when a service supports event stream operations?

Reply:
Seems reasonable.

Reply:
If that's the only type for which a consumer will have to take a dependency on `aws-smithy-types` then we might as well export that too. Unless, that involves exporting a lot more stuff from the `aws-smithy-types::strbytes` mod.

Reply:
`StrBytes` added in reexports in c8c293b.